tests/micro_benchmarks/benchmark_row_delete.cpp

```
@@ -0,0 +1,134 @@
+		ASSERT(i < n);
+		info = Delta_MatrixTupleIter_iterate_range(&it, i, i + batch_size);
+		i += batch_size;
+
+		n_tups = 0;
+		while (info == GrB_SUCCESS) {
+			GrB_Index r, c;
+			info = Delta_MatrixTupleIter_next_BOOL(&it, &r, &c, NULL);
+			tuples[n_tups++] = r;
+			tuples[n_tups++] = c;
+		}
+		ASSERT(info >= 0);
```

Cppcheck (2.19.0)

[error] Null pointer dereference

(nullPointer)